diff --git a/cmake/CMakeLists.txt b/cmake/CMakeLists.txt
index 1bcb0d8677..542fbe824a 100644
--- a/cmake/CMakeLists.txt
+++ b/cmake/CMakeLists.txt
@@ -274,6 +274,7 @@ install(TARGETS lmp EXPORT LAMMPS_Targets DESTINATION ${CMAKE_INSTALL_BINDIR})
 option(CMAKE_VERBOSE_MAKEFILE "Generate verbose Makefiles" OFF)
 
 set(STANDARD_PACKAGES
+  SAMQEQ
   ADIOS
   AMOEBA
   APIP
@@ -456,6 +457,7 @@ endif()
 pkg_depends(ML-IAP ML-SNAP)
 pkg_depends(LATBOLTZ MPI)
 pkg_depends(SCAFACOS MPI)
+pkg_depends(SAMQEQ KSPACE)
 pkg_depends(AMOEBA KSPACE)
 pkg_depends(DIELECTRIC KSPACE)
 pkg_depends(DIELECTRIC EXTRA-PAIR)
